berkdb/db/db_meta.c

```diff
@@ -620,10 +620,10 @@ __db_new_original(dbc, type, pagepp)
 			if (gbl_core_on_sparse_file) {
 				char cmd[100];
 
-				snprintf(cmd, sizeof(cmd), "gcore %d",
-				    getpid());
+				snprintf(cmd, sizeof(cmd), "gcore %d", getpid());
 				printf("%s\n", cmd);
-				system(cmd);
+				int lrc = system(cmd);
```

Comment:
Maybe just (void)system(cmd) ...

Reply:
i tried that and did not silence that warning.

Reply:
Tried again, alas does not like (void):
/comdb2/tests/tools/nemesis.c:234:9: error: ignoring return value of 'system', declared with attribute warn_unused_result [-Werror=unused-result]
         (void)system(cmd);
         ^